osm2pgsql.lua

```diff
@@ -660,6 +693,13 @@ function osm2pgsql.process_relation(object)
         end
     end
 
+-- Adds tags to turn off Israel admin 4 boundaries for Palestine.
+    if type == 'boundary' and (object.tags.admin_level == '4') and object.tags['ISO3166-2'] then
+        if osm2pgsql.has_prefix(object.tags['ISO3166-2'], 'IL-') then
+            output_hstore['admin_level:PS'] = '6'
```

Comment:
There are likely more countries this should apply to, coordinate with @jeffdefacto

Reply:
Cyprus has this done with an actual relation and Taiwan has similar logic above. Are there additional countries where this would apply?